test/test_overrides.py

```diff
@@ -678,5 +758,47 @@ def unwrap(v):
                                        torch.nn.functional.bilinear(a, c, b)))
 
 
+class TestGradCheckOverride(TestCase):
+    "Test that wrappers work with gradcheck."
+    def test_gradcheck(self):
+        from torch.autograd import gradcheck
+
+        a = wrap(torch.tensor(5.0, dtype=torch.double))
+        b = wrap(torch.tensor(6.0, dtype=torch.double))
+
+        a.requires_grad = True
+        b.requires_grad = True
+
+        gradcheck(torch.add, (a, b), raise_exception=False)
+
+        total_used_attrs = a.used_attrs.union(b.used_attrs)
+        total_used_calls = a.used_calls.union(b.used_calls)
+
+        self.assertEqual(total_used_attrs, {
```

Comment:
It's probably worth pointing out that it's OK for the total used attributes to wobble if people make changes to gradcheck, and maybe give a little guidance about what differences are ok and what are not.